src/sentry/static/sentry/app/views/projectInstall/platform.tsx

```diff
@@ -166,7 +166,7 @@ class ProjectInstallPlatform extends React.Component<Props, State> {
                   <React.Fragment>
                     {showPerformancePrompt && (
                       <Feature
-                        features={['organization:performance-view']}
+                        features={['performance-view']}
```

Comment:
I think you could have kept the prefix, it just would be `organizations:`

Reply:
Unprefixed seems to be how most of the features are defined and how I'll do them in the future as to make fewer mistakes :smile:

Reply:
👍 

I think the prefix only exists for collisions with `projects` and `organizations`